lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -68,20 +68,15 @@ def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler
               add_wording_offense(description_node, MSG_SHOULD)
             elsif message.match?(IT_PREFIX)
               add_wording_offense(description_node, MSG_IT)
-            else
-              check_and_handle_insufficient_examples(description_node)
+            elsif insufficient_docstring?(description_node)
+              add_offense(docstring(description_node),
```

Comment:
How about adding `next if text(node) == replacement_text(node)` in the auto-correction?
RuboCop thinks correction was made because the autocorrect didn't return `nil`, even though the correction resulted in replacing with the same value

Reply:
It feels a bit hacky, for two reasons.
For "should do something" we know we have a correction, but we'll be making this check that will also return `false`.
For "works" we know we don't have a correction, so the check will be `true`.
I'm shortcutting (in a good way) here to skip autocorrection for "works", along with this check.